### ISSUE TITLE
No cron running?

### ISSUE THREAD

**Issue by Altalus** [itflow-org/itflow-docker#7]
Hi, great to see this as a docker container, makes my life a _lot_ easier for demo to the bosses, but I seem unable to make the email part work. The email queue stays at "Queued" and nothing seems to be processed.

I made sure to go to the Notification and enable cron. I see no logs at all in "App Logs".

Any idea ? :)

**Comment by Altalus** [comment]
I checked and yeah, cron is not started automatically in the container and no itflow cron jobs added. I had to do it manually.

**Comment by NXT44** [comment]
Can you explain how to do this please.
I have the same issue.

**Comment by Altalus** [comment]
@wrongecho sure, I can understand that, I do not have a VM I can install it into at the moment unfortunately and the docker solution was perfect for me. I'll see what I can do, I might need to check if I can setup one and I'll PR my solution if it aligns with you :)

**Comment by NXT44** [comment]
Added the lines in the crontab started it manually but still in the queue

**Comment by Altalus** [comment]
@NXT44 yes, I am troubleshooting this myself. I'll let you know if I can find a workaround.

**Comment by magixnetworks** [comment]
I have updated the README with some additional information on setting up Cron as it can't run as the www-data user like the non-docker version does.

**Comment by magixnetworks** [comment]
Cron works correctly and is configured automatically on the latest docker build (:latest tag)
